rustfmt.toml

```diff
@@ -0,0 +1,2 @@
+reorder_imports = true
```

Comment:
he's biased it seems :P ..
It seems OK though, no ? It will make things uniform.